contracts/src/access/test/mocks/contracts/AccessControl.mock.compact

```diff
@@ -5,17 +5,19 @@ pragma language_version >= 0.18.0;
 
 import CompactStandardLibrary;
 
-import "../AccessControl" prefix AccessControl_;
+import "../../../AccessControl" prefix AccessControl_;
 
 // Export circuits for testing purposes
 export { AccessControl_hashUserRole,
          AccessControl_roleCommits,
          AccessControl_isInitialized,
-         AccessControl_index };
+         AccessControl_index,
+         AccessControl_Role,
+         ZswapCoinPublicKey };
 
 /**
  * @description Constructs the contract by initializing the AccessControl module with an initial admin.
- * @param {ZswapCoinPublicKey} initialAdmin - The public key of the initial admin.
+ * @param {AccessControl_ZswapCoinPublicKey} initialAdmin - The public key of the initial admin.
```

Comment:
What's the benefit of including the prefix? It'd make sense if it was a user-defined type, but irrespective of where `ZswapCoinPublicKey` comes from, it's the same compact std type